addons/SimpleTODO/TODOColumn.gd

```diff
@@ -1,27 +1,79 @@
 tool
 extends PanelContainer
 
-onready var name_edit = $VBoxContainer/HBoxContainer2/Name
+onready var head = $VBoxContainer/Head
+onready var top_separator = $VBoxContainer/TopSeparator
+onready var bottom_separator = $VBoxContainer/BottomSeparator
+onready var scroll_container = $VBoxContainer/ScrollContainer
+onready var actions = $VBoxContainer/Actions
+onready var minimize_button = $VBoxContainer/Head/Minimize
+
+onready var name_edit = $VBoxContainer/Head/Name
 onready var item_container = $VBoxContainer/ScrollContainer/Items
-onready var counter = $VBoxContainer/HBoxContainer2/Counter
-onready var delete_button = $VBoxContainer/HBoxContainer/DeleteColumn
+onready var counter = $VBoxContainer/Head/Counter
+onready var delete_button = $VBoxContainer/Actions/DeleteColumn
 onready var timer = $Timer
 
 var undo_redo: UndoRedo
+var minimized = false setget set_minimized
+var main: Control
+
+var item_placement_holder: Panel
+var is_dragging = false
+var initial_item_index = 0
+var current_drag_item_index = 0
+
+var item_margin = 20
+
+func set_minimized(val):
+	minimized = val
+	
+	var new_text = "V"
+	
+	if not val:
+		new_text = "^"
```

Comment:
I'd replace the text with an icon from editor icons; it's look better.

Reply:
Agreed, though any hints on how I would grab an editor icon from code?

Reply:
`get_icon(icon name, "EditorIcons")`
There's a plugin called Editor Icons Previewer in the asset lib that can help you pick a good icon.